.gitignore

```diff
@@ -78,3 +78,4 @@ lockfiles/
 
 # custom CI setup 
 report.json
+Pipfile
```

Comment:
Must: This shouldn't be needed, we stopped using `Pipfile`s a long time ago. Please use the devcontainer to develop with dodal if at all possible, happy to help you set up your dev environment.